openedx/features/edly/tests/test_utils.py

```diff
@@ -2,24 +2,37 @@
 Tests for Edly Utils Functions.
 """
 import jwt
+import mock
 from mock import MagicMock
 
 from django.conf import settings
 from django.http import HttpResponse
 from django.test import TestCase
 from django.test.client import RequestFactory
 
+from openedx.core.djangolib.testing.utils import skip_unless_cms
 from openedx.features.edly import cookies as cookies_api
 from openedx.features.edly.tests.factories import EdlySubOrganizationFactory, EdlyUserProfileFactory, SiteFactory
 from openedx.features.edly.utils import (
     create_user_link_with_edly_sub_organization,
     decode_edly_user_info_cookie,
     encode_edly_user_info_cookie,
     get_edly_sub_org_from_cookie,
-    user_has_edly_organization_access
+    get_edx_org_from_cookie,
+    set_global_course_creator_status,
+    update_course_creator_status,
+    user_has_edly_organization_access,
+)
+from student import auth
+from student.roles import (
+    CourseCreatorRole,
+    GlobalCourseCreatorRole
```

Comment:
nit: comma at the end of last item in list or dict in python

Reply:
done